packages/dataviews/README.md

```diff
@@ -275,10 +275,6 @@ Whether the data is loading. `false` by default.
 
 Array of layouts supported. By default, all are: `table`, `grid`, `list`.
 
-### `deferredRendering`: `boolean`
```

Comment:
As I mentioned, I think we should keep this mode for now, until we've polished the interactivity to a level we're comfortable with. If you all still think we should remove it this now, we'd need to add a Changelog note.

Reply:
Restoring this won't make a difference. All previews are already deferred, so there's nothing else to defer other than rendering the bare grid

Reply:
My point was to keep this in the API for now, but not use it until we improve the interactivity. If we are doing that _before_ merging this PR, I'm fine removing it.

Reply:
Interactivity is now on pair with `trunk` so we can remove this.